sgf-viewer/src/content.ts

```diff
@@ -1079,16 +1098,50 @@ export const pages: object =
         ],
         "content": [
             {
-                "title": "Strength evaluation",
-                "dir_name": "vit-strength",
+                "title": "Professional game: Yilun Yang",
+                "dir_name": "vit-vs-yang",
+                "paths": ["/shared/sgf-viewer-games/vit-vs-yang.sgf"],
+                "max_games": 1,
+                "adversary": "Yilun Yang",
+                "victim": "<code>ViT-victim</code>, 65536 visits, 64 search threads",
+                "description": [
+                    "After a close early game between ViT (black) and Yang (white), a complicated fight develops on the left side and center, starting around <a class='clickable' onclick='setMove(`vit-vs-yang`, 52)'>move 52</a>. According to KataGo, this move would have been better as a cap two lines to the right of black's stone on the middle of the left side, or one line below the cap. If so, the game would remain even, even slightly advantageous for white. Also, black <a class='clickable' onclick='setMove(`vit-vs-yang`, 55)'>55</a> was a mistake and should have been the cut where white played <a class='clickable' onclick='setMove(`vit-vs-yang`, 56)'>56</a>; if now instead of connecting at <a class='clickable' onclick='setMove(`vit-vs-yang`, 56)'>56</a> white pressures black's stones above by playing one line above and to the left of the connection, white will again have a very slight advantage.",
+                    "Nonetheless, after these moves the game remained close. The most deciding move was that White should have played <a class='clickable' onclick='setMove(`vit-vs-yang`, 66)'>66</a> directly below white's leftmost stone, to probe how black connects the two stones above. If black plays the best, empty triangle connection, then a complicated ko situation may develop but the game remains only marginally favored for black, whereas other connections will tilt the game in white's favor. But after <a class='clickable' onclick='setMove(`vit-vs-yang`, 66)'>66</a> in the game, black threatens to cut off a stone with <a class='clickable' onclick='setMove(`vit-vs-yang`, 67)'>67</a>, and the timing to probe the connection on the left is lost. In the end, with <a class='clickable' onclick='setMove(`vit-vs-yang`, 92)'>92</a> white is able to live, but black's position on the outside is better and black has a decent advantage.",
```

Comment:
We don't capitalize white elsewhere so suggest not capitalizing it here for consistency (or change it everywhere)
```suggestion
                    "Nonetheless, after these moves the game remained close. The most deciding move was that white should have played <a class='clickable' onclick='setMove(`vit-vs-yang`, 66)'>66</a> directly below white's leftmost stone, to probe how black connects the two stones above. If black plays the best, empty triangle connection, then a complicated ko situation may develop but the game remains only marginally favored for black, whereas other connections will tilt the game in white's favor. But after <a class='clickable' onclick='setMove(`vit-vs-yang`, 66)'>66</a> in the game, black threatens to cut off a stone with <a class='clickable' onclick='setMove(`vit-vs-yang`, 67)'>67</a>, and the timing to probe the connection on the left is lost. In the end, with <a class='clickable' onclick='setMove(`vit-vs-yang`, 92)'>92</a> white is able to live, but black's position on the outside is better and black has a decent advantage.",
```